pages/help/help_domain-requests.md

```diff
@@ -91,21 +81,20 @@ Follow [these steps to create your Login.gov account](https://login.gov/help/get
 You can make changes to your domain request after you submit it. To change your request you have to withdraw it, make your changes, and submit it again. Changing your request might add to the wait time for the .gov to review it.
 
 Follow these steps to make changes. 
-<ol>
-<li>Sign in with your Login.gov account to <a href="#">manage your domain</a>.</li> 
-<li>Click the “Manage” link for the domain you want to change. This takes you to the domain overview page.</li>
-<li>Click the “Withdraw request” button to withdraw your request.</li>
-<li>Confirm that you want to withdraw your request. This takes you back to the domain management page.</li>
-<li>Click the “Manage” link for the domain you want to change. This takes you to the domain overview page.</li>
-<li>Go to the section of the request you want to change. Make your changes.</li> 
-<li>Click “Submit.”</li>
-</ol>
+
+1. [Sign in with your Login.gov account](#).
+2. Click the “Manage” link for the domain you want to change. This takes you to the domain overview page.
+3. Click the “Withdraw request” button to withdraw your request.
+4. Confirm that you want to withdraw your request. This takes you back to the domain management page.
+5. Click the “Manage” link for the domain you want to change. This takes you to the domain overview page.
+6. Go to the section of the request you want to change. Make your changes. 
+7. Click “Submit.”
 
 You will receive an email notification about your changes. 
 
 ## Check the status of your domain request
 
-You can check the status of your domain request at any time. Sign in with your Login.gov account to <a href="#">manage your domain</a>. The status of your domain request will be on this page under “Active domain requests.”
+You can check the status of your domain request at any time. <a href="#">Sign in with your Login.gov account</a>. This will take you to the domain management page. The status of your domain request will be on this page under “Active domain requests.”
```

Comment:
Can make this one a markdown style link

Reply:
Thanks! (Aside: I just rewrote it to align with the other new text.)